playbooks/bosh-acceptance-tests/run.yaml

```diff
@@ -0,0 +1,137 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: export-cloud-openrc
+      vars:
+        cloud_name: 'opentelekomcloud'
+  tasks:
+    - name: Run cloudfoundry acceptance tests of bosh-acceptance-tests
+      shell:
+        cmd: |
+          set -x
+          set -e
+          set -o pipefail
+
+          # Cloud provider specific variables
+          NET_ID="134ca339-24dc-44f5-ae6a-cf0404216ed2"
+          SUBNET_ID="3d6e9157-655f-42ec-9778-94d455730e5c"
+          SMALL_FLAVOR="s2.medium.1" #1U1G
+          MEDIUM_FLAVOR="s2.large.1" #2U2G
+          LARGE_FLAVOR="s2.2xlarge.1" #8U8G
+          BUILD_ID="{{ zuul.build }}"
+
+          pip install python-openstackclient
+          curl -Lo ./bosh https://s3.amazonaws.com/bosh-cli-artifacts/bosh-cli-3.0.1-linux-amd64
+          chmod +x ./bosh
+          cp ./bosh /usr/local/bin/bosh
+          bosh -v
+          apt-get install -y build-essential zlibc zlib1g-dev ruby ruby-dev openssl libxslt-dev libxml2-dev libssl-dev libreadline6 libreadline6-dev libyaml-dev libsqlite3-dev sqlite3
+
+          mkdir bosh-1 && cd bosh-1
+          git clone https://github.com/cloudfoundry/bosh-deployment.git
+          openstack keypair create microbosh-${BUILD_ID} > microbosh.pem
+          openstack security group create bosh-${BUILD_ID} --description 'BOSH Security Group'
+          openstack security group rule create --protocol tcp --ingress bosh-${BUILD_ID}
+          openstack security group rule create --protocol icmp --ingress bosh-${BUILD_ID}
+
+          wget --content-disposition https://bosh.io/d/stemcells/bosh-openstack-kvm-ubuntu-trusty-go_agent -O bosh-openstack-kvm-ubuntu-trusty-go_agent.tgz
+          sed -i 's|https://bosh.io/d/stemcells/.*$|file://{{ ansible_user_dir }}/bosh-1/bosh-openstack-kvm-ubuntu-trusty-go_agent.tgz|' bosh-deployment/openstack/cpi.yml
+          sed -i "s/m1.small/${SMALL_FLAVOR}/g" `grep m1.small -rl bosh-deployment/openstack`
+          sed -i "s/m1.xlarge/${LARGE_FLAVOR}/g" `grep m1.xlarge -rl bosh-deployment/openstack`
+          sed -i '/^.*human_readable_vm_names.*$/ a\ \ \ \ state_timeout: 300000' bosh-deployment/openstack/cpi.yml
+
+          apt-get install ruby ruby-dev -y
+
+          _port_id=$(openstack port create --network ${NET_ID} bosh-port -f value -c id)
```

Comment:
Please add some NOTES about why we create port, then delete it immediately.

Reply:
done